test/integration/BrandedToken/RejectStakeRequest.js

```diff
@@ -32,10 +32,9 @@ const Setup = Package.EconomySetup;
 const { assert } = chai;
 const config = require('../../utils/configReader');
 const StakeHelper = require('../../../lib/helpers/stake/gateway_composer/StakeHelper');
-const Staker = require('../../../lib/helpers/stake/gateway_composer/Staker');
+const Staker = require('../../../lib/Staker');
```

Comment:
I think in the tests it should get Staker class from Package(index.js)
const Staker = Package.Helpers.Staker